hoff.cabal

```diff
@@ -63,6 +62,7 @@ library
                , wai
                , warp
                , warp-tls
+  other-modules: Paths_hoff
```

Comment:
When adding it you should also add it under `autogen-modules`, _in addition to_ `other-modules`, because you want Cabal to know that the haskell source it looks at is created by another program (in this case that other program is also Cabal)

Reply:
Done!  :runner: